libzeth/tests/circuits/binary_operation_test.cpp

```diff
@@ -11,36 +11,36 @@
 
 using namespace libzeth;
 
-typedef libff::default_ec_pp ppT;
-typedef libff::Fr<ppT> FieldT;
+using ppT = libff::default_ec_pp;
+using FieldT = libff::Fr<ppT>;
 
 namespace
 {
 
 TEST(TestXOR, TestTrue)
 {
     libsnark::protoboard<FieldT> pb;
-    libsnark::pb_variable<FieldT> ZERO;
-    ZERO.allocate(pb, "zero");
-    pb.val(ZERO) = FieldT::zero();
+    libsnark::pb_variable<FieldT> zero;
+    zero.allocate(pb, "zero");
+    pb.val(zero) = FieldT::zero();
 
     libsnark::pb_variable_array<FieldT> a = variable_array_from_bit_vector(
         {
-            0, 0, 0, 0, 1, 1, 1, 1, // 0F
-            0, 1, 0, 1, 0, 1, 0, 1, // 55
-            1, 0, 1, 0, 1, 0, 1, 0, // AA
-            1, 1, 1, 1, 0, 0, 0, 0  // F0
+            false, false, false, false, true,  true,  true,  true,  // 0F
```

Comment:
The check `readability-implicit-bool-conversion` will need to be either de-activated or `// NOLINT` comments will be needed to make sure that our arrays of booleans can continue to be expressed as 0's and 1's